test/SlowTests/Issues/RavenDB_2435.cs

```diff
@@ -1,121 +0,0 @@
-// -----------------------------------------------------------------------
-//  <copyright file="RavenDB_2435.cs" company="Hibernating Rhinos LTD">
-//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
-//  </copyright>
-// -----------------------------------------------------------------------
-
-using System;
-using System.Linq;
-using FastTests;
-using Raven.Client.Documents;
-using Raven.Client.Documents.Indexes;
-using Raven.Client.Documents.Linq.Indexing;
-using Raven.Client.Documents.Transformers;
-using Raven.Tests.Core.Utils.Entities;
-using Xunit;
-
-namespace SlowTests.Issues
-{
-    public class RavenDB_2435 : RavenTestBase
-    {
-        private class Index1 : AbstractIndexCreationTask<Person>
-        {
-            public Index1()
-            {
-                Map = people => from person in people
-                                select new
-                                {
-                                    Name = person.Name.StripHtml()
```

Comment:
Restore candidate: it verifies `DynamicExtensionMethods.StripHtml` is working during _indexing_